kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -362,6 +365,25 @@ protected void close() {
                 currentConnectedGroup.remove(clientHost);
             }
 
+            // Try to remove all stored groupID on the metadata store.
+            if (log.isDebugEnabled()) {
+                log.debug("Try to remove all stored groupID on the metadata store. Current connected clientIds: {}",
+                        currentConnectedClientId);
+            }
+            currentConnectedClientId.forEach(clientId -> {
+                String path = groupIdStoredPath + GroupIdUtils.groupIdPathFormat(clientHost, clientId);
+                metadataStore.deleteRecursive(path)
+                        .whenComplete((__, ex) -> {
+                            if (ex != null) {
+                                log.error("Delete groupId failed. Path: [{}]", path, ex);
+                                return;
+                            }
```

Comment:
We should ignore the `MetadataStoreException$NotFoundException`. Here are some logs when I test this patch in a standalone.

```
2022-04-28T15:06:13,143+0800 [metadata-store-22-1] ERROR io.streamnative.pulsar.handlers.kop.KafkaRequestHandler - Delete groupId failed. Path: [/client_group_id/127.0.0.1-consumer-my-group-3]
java.util.concurrent.CompletionException: org.apache.pulsar.metadata.api.MetadataStoreException$NotFoundException: org.apache.zookeeper.KeeperException$NoNodeException: KeeperErrorCode = NoNode for /client_group_id/127.0.0.1-consumer-my-group-3
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292) ~[?:1.8.0_332]
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308) ~[?:1.8.0_332]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:714) ~[?:1.8.0_332]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:701) ~[?:1.8.0_332]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488) ~[?:1.8.0_332]
	at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990) ~[?:1.8.0_332]
	at org.apache.pulsar.metadata.impl.ZKMetadataStore.lambda$null$10(ZKMetadataStore.java:341) ~[org.apache.pulsar-pulsar-metadata-2.10.0.jar:2.10.0]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_332]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_332]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_332]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_332]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_332]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_332]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [io.netty-netty-common-4.1.74.Final.jar:4.1.74.Final]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_332]
Caused by: org.apache.pulsar.metadata.api.MetadataStoreException$NotFoundException: org.apache.zookeeper.KeeperException$NoNodeException: KeeperErrorCode = NoNode for /client_group_id/127.0.0.1-consumer-my-group-3
	at org.apache.pulsar.metadata.impl.ZKMetadataStore.getException(ZKMetadataStore.java:428) ~[org.apache.pulsar-pulsar-metadata-2.10.0.jar:2.10.0]
	... 9 more
Caused by: org.apache.zookeeper.KeeperException$NoNodeException: KeeperErrorCode = NoNode for /client_group_id/127.0.0.1-consumer-my-group-3
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:118) ~[org.apache.zookeeper-zookeeper-3.6.3.jar:3.6.3]
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:54) ~[org.apache.zookeeper-zookeeper-3.6.3.jar:3.6.3]
	at org.apache.pulsar.metadata.impl.ZKMetadataStore.getException(ZKMetadataStore.java:422) ~[org.apache.pulsar-pulsar-metadata-2.10.0.jar:2.10.0]
	... 9 more
```